apps/general/models.py

```diff
@@ -1,3 +1,230 @@
+    name = models.CharField(max_length=255, null=False)
+    print_name = models.CharField(max_length=255, null=True, default=None)
+    short_name = models.CharField(max_length=100, null=True, default =None)
+    code = models.CharField(max_length=100, null=True, default=None)
+    num_branches = models.IntegerField(default=0)
+    logo = models.ImageField(null=True, upload_to=company_logos, default=None)
+    address = models.TextField(null=True)
+    country = models.CharField(max_length=100,null=True, default=None)
+    state = models.CharField(max_length=255, null=True, default=None)
+    city = models.CharField(max_length=255, null=True, default=None)
+    pin_code = models.CharField(max_length=20, null=True, default=None)
+    phone_regex = RegexValidator(regex=r'^\+?1?\d{9,15}$', message="Phone number must be entered in the format: '+999999999'. Up to 15 digits allowed.")
+    phone = models.CharField(validators=[phone_regex], max_length=20, null=True, default=None)
+    email = models.EmailField(max_length=255, null=True)
+    longitude = models.DecimalField(max_digits=9, decimal_places=6, null=True, default=None)
+    latitude = models.DecimalField(max_digits=9, decimal_places=6, null=True, default=None)
+    print_address = models.TextField(null=True, default=None)
+    website = models.URLField(max_length=255, default=None, null=True)
+    facebook_url = models.URLField(max_length=255, default=None, null=True)
+    skype_id = models.CharField(max_length=50, default=None, null=True)
+    twitter_handle = models.CharField(max_length=50, default=None, null=True)
+    linkedin_url = models.URLField(max_length=255, default=None, null=True)
+    pan = models.CharField(max_length=50, default=None, null=True)
+    tan = models.CharField(max_length=50, default=None, null=True)
+    cin = models.CharField(max_length=50, default=None, null=True)
+    gst_tin = models.CharField(max_length=50, default=None, null=True)
+    establishment_code = models.CharField(max_length=50, default=None, null=True)
+    esi_no = models.CharField(max_length=50, default=None, null=True)
+    pf_no = models.CharField(max_length=50, default=None, null=True)
+    authorized_person = models.CharField(max_length=255, default=None, null=True)
+    iec_code = models.CharField(max_length=50, default=None, null=True)
+    eway_username = models.CharField(max_length=100, default=None, null=True)
+    eway_password = models.CharField(max_length=100, default=None, null=True)
+    gstn_username = models.CharField(max_length=100, default=None, null=True)
+    gstn_password = models.CharField(max_length=100, default=None, null=True)
+    VAT_GST_STATUS_CHOICES = (
+        ('Active', 'Active'),
+        ('Inactive', 'Inactive'),
+        ('Pending', 'Pending'),
+    )
+    vat_gst_status = models.CharField(max_length=10, choices=VAT_GST_STATUS_CHOICES, default=None, null=True)
+    GST_TYPE_CHOICES = (
+        ('Goods', 'Goods'),
+        ('Service', 'Service'),
+    )
```

Comment:
gst_type has Both option also which is missing here

Reply:
Yeah Thulasi I will add this also.